types/stripe/index.d.ts

```diff
@@ -63,6 +63,16 @@ interface ResponseEvent {
     request_end_time: number;
 }
 
+interface StripeConfig {
+    apiVersion: string;
```

Comment:
From the doc it looks like all the properties here should be marked optional.  As is all properties have to be specified.

`apiVersion` and `httpAgent` also allow null so their types should be `string | null` and `Agent | null` respectively.